src/breadcrumbs/breadcrumbs.scss

```diff
@@ -21,45 +21,101 @@
 
 @mixin iui-breadcrumbs-item {
   display: flex;
+  align-items: center;
   line-height: $iui-component-height;
   height: $iui-component-height;
-  max-width: $iui-3xl * 2;
-  margin: 0 $iui-sm;
 
-  a {
-    @include iui-anchor;
-  }
-
-  .iui-button {
-    @include iui-button-borderless;
-    margin: 0 (-$iui-s - 1);
+  > * {
+    max-width: 26ch;
   }
+}
 
-  > *,
-  .iui-button-label {
+@mixin iui-breadcrumbs-item-overrides {
+  > * {
+    padding: 0 $iui-s;
     overflow: hidden;
     white-space: nowrap;
     text-overflow: ellipsis;
+    @include themed {
+      color: t(iui-text-color);
+    }
   }
 
-  &:not(.iui-current) > .iui-button {
-    @include themed {
-      color: t(iui-color-foreground-primary);
+  > :any-link {
+    @include iui-anchor;
+
+    &:focus {
+      outline-offset: -1px;
+    }
+  }
+
+  .iui-button {
+    &.iui-button {
```

Comment:
This is what is causing the 0,5,0.  We need to increase the specificity so that the borderless mixin overrides a button with `.iui-default`.  If you apply `.iui-default` to the buttons on [the live breadcrumb demo](https://itwin.github.io/iTwinUI/backstop/minified/breadcrumbs.html), we get this:
![Screen Shot 2022-05-04 at 4 44 04 PM](https://user-images.githubusercontent.com/849817/166822760-07986e52-179e-4c38-8acc-5184fd02766e.png)

I'm not sure how we're able to use `.iui-default` in React.

Reply:
Vykintas and I discussed this, seems like the CSS is being imported in a different order in React than in this repo.  In our live demo we were not including the `.iui-default` modifier on our buttons, so they appeared ok.  When adding the modifier class using dev tools they appear incorrectly.  In this PR I am adding the modifier class AND increasing the specificity to 0,5,0 so the import order does not matter.

The only good news is that this older method can be removed in React 2.0.